specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/createOrUpdateCommandJob.json

```diff
@@ -0,0 +1,102 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "workspace-1234",
+    "workspaceName": "testworkspace",
+    "api-version": "2020-09-01-preview",
+    "id": "testId",
+    "body": {
+      "properties": {
+        "jobType": "Command",
+        "codeConfiguration": {
+          "codeArtifactId": "some code artifact",
+          "command": [
+            "python",
+            "train.py",
```

Comment:
>train.py", [](start = 13, length = 10)

How are these scripts put in the workspace?

Reply:
CodeArtifactId refers to a CodeVersion: https://github.com/Azure/azure-rest-api-specs/pull/10876

CodeVersion manages files (via blob url). You can create a CodeVersion as a blob file or blob directory. In the Job, you can reference these files.